Tests/Runner/RunnerTest.php

```diff
@@ -889,23 +695,15 @@ function ($a, $b) {
         self::assertEquals('fooBar1', $row1['state']['bazRow1']);
         self::assertArrayHasKey('bazRow2', $row2['state']);
         self::assertEquals('fooBar2', $row2['state']['bazRow2']);
+        self::assertTrue($this->getRunnerHandler()->hasInfoThatContains('Running component keboola.docker-demo-sync (row 1 of 2)'));
+        self::assertTrue($this->getRunnerHandler()->hasInfoThatContains('Running component keboola.docker-demo-sync (row 2 of 2)'));
+        self::assertTrue($this->getRunnerHandler()->hasInfoThatContains('Waiting for 2 storage jobs'));
```

Comment:
`Storage`, velké `S`?

Reply:
spravim to pozdeji https://github.com/keboola/docker-bundle/issues/385